apps/explorer/src/pages/epochs/stats/EpochProgress.tsx

```diff
@@ -25,15 +25,32 @@ export function EpochProgress({
 }: EpochProgressProps) {
     const { progress, label } = useEpochProgress();
 
+    const elapsedTime =
+        !inProgress && start && end ? getElapsedTime(start, end) : undefined;
+
     return (
         <Card bg={inProgress ? 'highlight' : 'default'} spacing="lg">
             <div className="flex min-w-[136px] flex-col space-y-16">
-                <div className="space-y-4">
-                    <Heading color="steel-darker" variant="heading3/semibold">
-                        {inProgress
-                            ? `Epoch ${epoch} in progress`
-                            : `Epoch ${epoch}`}
-                    </Heading>
+                <div className={clsx(inProgress ? 'space-y-4' : 'space-y-6')}>
+                    <div className="flex flex-col gap-2">
+                        <Heading
+                            color="steel-darker"
+                            variant="heading3/semibold"
+                        >
+                            {inProgress
+                                ? `Epoch ${epoch} in progress`
+                                : `Epoch ${epoch}`}
+                        </Heading>
+                        {elapsedTime && (
+                            <Heading
+                                variant="heading6/medium"
+                                color="steel-darker"
+                            >
+                                {elapsedTime.hours}h {elapsedTime.minutes}m{' '}
```

Comment:
it might be a little cleaner to use one interpolated template string but this no big deal

Reply:
Good call, I will just make the function returning the string directly